app/cp_nigeria/helpers.py

```diff
@@ -71,6 +71,17 @@ def csv_to_dict(filepath, label_col="label"):
 OUTPUT_PARAMS = csv_to_dict("cpn_output_params.csv")
 
 
+def calculate_co2_mitigation(project):
+    # TODO implement the calculation
+    co2_mitigations = {"Ebute-Ipare": 6894, "Usungwe": 22123, "Unguwar Kure": 3088, "Ezere": 8981, "Egbuniwe": 7472}
+
+    for key in co2_mitigations.keys():
+        if project.name or project.options.community.name in key:
+            return f"{co2_mitigations[key]:,}"
+
+    return "[not available]"
```

Comment:
I personally dislike multiple return within one function, but I looked for best practices and couldn't find enough backing arguments, so I let you choose :)

My main argument is to debug : I find it easier to look at the value of the `answer` or `result` variable towards the end of the function otherwise one does not know easily at which `return` the function ended and thus in which state it was

Reply:
Thanks! I think that makes sense. Left this as is for now as I will implement this function properly within the next week anyways, then I will change it to an `answer` variable instead :)